### ISSUE TITLE
Login/ Register screen  S1 S2

### ISSUE THREAD

**Issue by mlapresta** [FondApp/Fond#5]
References: 
![image](https://user-images.githubusercontent.com/40969859/99136232-737b4c00-25e9-11eb-88a3-ec1980ae338f.png)

Acceptance criteria:
- [x] Screen should match wireframe 
- [x] Sign up should work (depends on #3 )
- [x] Login should work (depends on #4 )

**Comment by kat-kime** [comment]
Hi @mlapresta -
It looks like part of the acceptance criteria is to ensure that the:

- Sign up button works
- Login button works

Does this mean that I should not submit a pull request until ticket #3 and ticket #4 is complete?

Another question:
One dependency for completing this task is to insert our Logo.

- Do we have this asset?
- If not, is it okay if I include a placeholder for now?

**Comment by kat-kime** [comment]
Got feedback from Mae:

> I would say that you’re fine with either way, but I would lean more towards making a PR now so that we can see if the code review assignment actually works

**Comment by mlapresta** [comment]
Since it'd be easier for the reviewer to test, I would normally lean towards waiting...however you would be doing our first PR and I am curious as to whether the code review is set up correctly. I would submit it as a PR now since you're so ahead of things (🥇 !!)  and we'll be able to make sure code review works correctly sooner rather than later

**Comment by kat-kime** [comment]
@mlapresta -
Sounds good! I'll submit a PR meow.

**Comment by mlapresta** [comment]
@kat-kime  Also we don't have the asset currently, let's discuss on slack, but we can make the top bar a separate ticket so I would leave it out

**Comment by kat-kime** [comment]
@mlapresta So, I actually left the placeholder in for the sake of creating a PR, but I'm thinking I actually don't want to remove it until we have an asset on hand, as I'm using a Relative Layout (and it's a pretty big focal piece).

Do you mind if I just leave it for now? Once we have the asset, it will just require the person to switch images.

**Comment by mlapresta** [comment]
Placeholder is totally good! :)

**Comment by kat-kime** [comment]
Sounds good, thanks!
